### DISCUSSION TITLE
Close Window Button Electron Not Closing

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#9234] [CLI - Electron mode]
I've created an Electron app using quasar. When i run in dev mode 'quasar dev -m electron' the close button appears to do nothing in the main application Window as shown below. Why is this and how do i fix it? It happens even when i first make a vanilla project. 

![image](https://user-images.githubusercontent.com/1416749/117815630-ac604a80-b233-11eb-83c7-ac02c01074ff.png)

**Comment by hawkeye64** [comment]
@JokerMartini https://stackoverflow.com/a/31174463/5359558

**Reply by JokerMartini** [reply]
I'm not setting the window to frameless. I'm using the default project settings and everything. This same configuration works with Vuetify but does not work when im using quasar.

**Reply by hawkeye64** [reply]
@JokerMartini Maybe something is missing? Do a comparison and maybe a PR if so.

**Reply by rstoenescu** [reply]
@JokerMartini can you provide a barebones repo which reproduces this? Also, what OS are you using? There is no reason why this wouldn't work with Quasar. Electron is still in charge of the window. Also, what Electron version are you using?

**Reply by JokerMartini** [reply]
I found out the issue. This issue only persists when adding `autoHideMenuBar: true` to the **electron-main.js** file in order to remove the menuBar from the application. This prevents the user from having the ability to click the Close button in the menuBar at when running > quasar dev -m electron

**Reply by rstoenescu** [reply]
More info on frameless electron windows: https://next.quasar.dev/quasar-cli/developing-electron-apps/frameless-electron-window
